calculator.js

```diff
@@ -106,25 +106,26 @@ addingButton.addEventListener("click", function() {
     operator = "+";
     lastButtonClicked = "+";
     lastValue = result.textContent;
-    result.textContent = lastValue + lastButtonClicked;
+
 });
 subtractingButton.addEventListener("click", function() {
     operator = "-";
     lastButtonClicked = "-";
     lastValue = result.textContent;
-    result.textContent = lastValue + lastButtonClicked;
+    // lastButtonClicked = lastValue + "-";
```

Comment:
Why this commented not removed completely?

Reply:
As you saw better it is different this time  
I have removed it and pushed the update